src/pages/profile-page/profile-page.js

```diff
@@ -49,14 +49,12 @@ const ProfilePage = () => {
   }));
 
   const handleSaveUser = ({ firstName, lastName, email, phoneNumber, ...address }) => {
-    if (phoneNumber === null) {
-      phoneNumber = '';
-    }
+    const userPhoneNumber = phoneNumber || '';
```

Comment:
Why do you add this changes?

I think we already have it in scope of this PR:
https://github.com/ita-social-projects/horondi_client_fe/pull/2294/files

Reply:
Miscommunication, fixed